src/tests/calculate.test.js

```diff
@@ -0,0 +1,17 @@
+import calculate from '../logic/calculate';
+
+describe('unit tests for calculate.js', () => {
+  test('checking when = is clicked the result should be subtraction', () => {
+    const obj = {
+      total: 20,
+      next: 10,
+      operation: '-',
+    };
+    const buttonName = '=';
+    expect(calculate(obj, buttonName)).toStrictEqual({
+      total: '10',
+      next: null,
+      operation: null,
+    });
```

Comment:
- [ ] Please cover all the functionalities and scenarios of the calculate function. Or as much as you can think of. (Every operation, forbidden operations as division or modulus by 0, what happens if strings are sent? etc.

Making sure to cover every scenario when making tests is like creating a layer of protection to our code. If later on, we want to implement new functionalities to the calculator, we will know we're doing something wrong if out of a sudden tests start failing.